api/src/main/java/org/openapitools/openapistylevalidator/api/IRule.java

```diff
@@ -0,0 +1,14 @@
+package org.openapitools.openapistylevalidator.api;
+
+import java.util.List;
+import org.eclipse.microprofile.openapi.models.OpenAPI;
+import org.openapitools.openapistylevalidator.error.StyleError;
+
+public interface IRule {
+
+    String ruleName();
```

Comment:
remove "rule" because it's already the name of the interface. `name` will be enough.